### ISSUE TITLE
Rayyansaidi Developer

### ISSUE THREAD

**Issue by Ryyn-Sd** [rayyansaidi-com/app#7]
Can you put Rayyansaidi Developer on the app?

**Comment by nthnchu** [comment]
How would this work offline? Would we cache the website by having electron download it in the background with or just not provide it?

**Comment by Ryyn-Sd** [comment]
I was thinking we could use an html version.

**Comment by nthnchu** [comment]
With internet though I think an iframe is an easy way or we can alternatively use electron's `browserWindow.loadUrl();`.

**Comment by Ryyn-Sd** [comment]
offline

**Comment by Ryyn-Sd** [comment]
> With internet though I think an iframe is an easy way or we can alternatively use electron's `browserWindow.loadUrl();`.

How do we use the other one?

**Comment by Ryyn-Sd** [comment]
> With internet though I think an iframe is an easy way or we can alternatively use electron's [`browserWindow.loadUrl();`](https://www.electronjs.org/docs/api/browser-window#winloadurlurl-options).

I'll use <code>browserWindow.loadUrl();</code>

**Comment by Ryyn-Sd** [comment]
> > With internet though I think an iframe is an easy way or we can alternatively use electron's [`browserWindow.loadUrl();`](https://www.electronjs.org/docs/api/browser-window#winloadurlurl-options).
> 
> I'll use `browserWindow.loadUrl();`

**Comment by nthnchu** [comment]
```js
function createDeveloperWindow () {
  const developerWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: true
    }
  });
  developerWindow.loadUrl('https://developer.rayyansaidi.com/');
}
```
And when you want to open the developer site (for example onclick of a button or somthing):

```js
createDeveloperWindow();
```

**Comment by nthnchu** [comment]
Fixed in e00b0e4aca28cbfbb233febc40321556fcf56dc5
